Week 2 assignments

sachin286 opened the pull request.

sachin286 committed:
hover animation

sachin286 committed:
exercise-2.2 page creation

ShashankSuresh commented on Week-2/Exercise-2.2/index.html:
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
```
Please use a Prettifer plugin to intend the code.

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+*{
+	margin:0px;
```
Same here for indentation.

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+	margin: 0;
+	padding: 10px 80px 10px 80px;
+	background-size: cover;
+	font-family: Arial;
```
Please add this as variable

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+	float: right;
+}
+
+@media only screen and (max-width: 629px) {
```
Please add media queries to the end of the file

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+  text-align: center;
+  cursor: pointer;
+  outline: none;
+  color: #fff;
```
Move this to variables please.

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+.button {
+  margin-top:25px;
+  padding: 15px 25px;
+  font-size: 24px;
```
Move this to variables please.

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+
+.button:active {
+  background-color: #3e8e41;
+  box-shadow: 0 5px #666;
```
Move this to variables please.

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+  box-shadow: 0 9px #999;
+}
+
+.button:hover {background-color: #3e8e41}
```
Move this to variables please.

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+  cursor: pointer;
+  outline: none;
+  color: #fff;
+  background-color: #04AA6D;
```
Move this to variables please.

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+  background-color: #04AA6D;
+  border: none;
+  border-radius: 15px;
+  box-shadow: 0 9px #999;
```
Move this to variables please.

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,89 @@
+.button:hover {background-color: #3e8e41}
+
+.button:active {
+  background-color: #3e8e41;
```
Move this to variables please.

ShashankSuresh commented on Week-2/Exercise-2.1/style.css:
```diff
@@ -0,0 +1,86 @@
+:root {
```
Good work on the variables.

sachin286 committed:
updating post review comments

sachin286 commented:
I have updated the files on the variables and styling

ShashankSuresh commented on Week-2/Exercise-2.2/style.css:
```diff
@@ -0,0 +1,106 @@
+  --grey: #999;
+
+  --text-size: 24px;
+  --text-fam: Arial;
```
This can be renamed to `--font-family`

sachin286 committed:
update font variables

sachin286 commented:
variables names updated

ShashankSuresh commented:
1. Adding meta tags.
2. Responsiveness for mobile, tablet and desktop devices. 
3. Formatting (minor)
4. Attach screen-shots 
5. Beautification of form

ShashankSuresh commented on Week-2/Exercise-2.1/style.css:
```diff
@@ -0,0 +1,86 @@
+.main-wrapper:hover .div-two {
+    transform: rotate(90deg) translateY(-65px);
+    transition: all ease 1s;
+}
```
The rotation for Blue and Yellow is reversed. Kindly made the necessary modification.

sachin286 committed:
fixing the square transform axis

sachin286 committed:
updating review comments changes

sachin286 commented:
<img width="519" alt="Screenshot 2022-07-01 at 7 22 24 PM" src="https://user-images.githubusercontent.com/46752637/176908185-f73b8ba7-4f6c-4abf-a4c9-ef6cf21647d9.png">

updated the format and added the changes mentioned
